armi/settings/fwSettings/globalSettings.py

```diff
@@ -832,6 +833,14 @@ def defineSettings() -> List[setting.Setting]:
                 ]
             ),
         ),
+        setting.Setting(
+            CONF_USER_PLUGINS,
+            default=[],
+            label="User Plugins",
+            description="An optional place for users to define on-the-fly plugins "
+            "for things like one-off analysis.",
```

Comment:
add example of how this should be defined. like if my plugin lived in 
c:\mystuff\here\myPlugin.py
how does it expect it to be defined

Reply:
There are examples in the unit tests and at the top of this PR. 

I could put an example in the tutorial, sure.

Reply:
I can put some description of how to set the file paths in their, sure.

Reply:
Okay, I did both:

1. I improved this description, and
2. I added a basic guide to the docs.